extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchBuildBufferAggregator.java

```diff
@@ -181,4 +182,9 @@ private static int smear(int hashCode)
     return hashCode ^ (hashCode >>> 7) ^ (hashCode >>> 4);
   }
 
+  @Override
+  public void inspectRuntimeShape(RuntimeShapeInspector inspector)
+  {
+    inspector.visit("selector", selector);
```

Comment:
@AlexanderSaydakov could you review this? Do parameters of `HllSketch`, `lgK` and `tgtHllType` affect the runtime shape of `aggregate()`?

Reply:
If I recall the original design spec for monomorphic processing, missing some fields results in sub-optimal performance, but no worse than specifying no runtime shape. Is that still true, as in, could it be merged as is and confirmed later with no bad side-effect?

Reply:
@AlexanderSaydakov runtime shape could be intuited as the actual hot machine code path when executing the `@CalledFromHotLoop` method, `aggregate()` in this example. E. g. if `lgK` just participates some mathematical expressions, the hot machine code will be something like `mul %ax, %bx` no matter what value is in `lgK`. This means that `lgK` does *not* affect the runtime shape. On the contrary, if the internal logic is something like:
```java
if (lgK < 5) {
  execute some logic
} else {
  execute some entirely different logic
}
```
Then `lgK` does affect the hot code path, and therefore the runtime shape.

Reply:
@AlexanderSaydakov thank you. I included `lgK` in runtime shaped of both `HllSketchBuildBufferAggregator` and `HllSketchMergeBufferAggregator`, assuming that different paths exist in both `HllSketch.update()` and `Union.update()`. See https://github.com/apache/incubator-druid/pull/6893/commits/fb7ee63d9c0186c365a0b7b3d1c8cadc5e025168.